### ISSUE TITLE
Stored Object: paper:arxiv.2005.08513

### ISSUE THREAD

**Issue by mbahng** [mbahng/mbahng-com#66]
{
  "sourceId": "arxiv",
  "paperId": "2005.08513",
  "url": "https://arxiv.org/pdf/2005.08513",
  "title": "Convergence of constant step stochastic gradient descent for non-smooth non-convex functions",
  "authors": [
    "Pascal Bianchi",
    "Walid Hachem",
    "Sholom Schechtman"
  ],
  "abstract": "This paper studies the asymptotic behavior of the constant step Stochastic Gradient Descent for the minimization of an unknown function F , defined as the expectation of a non convex, non smooth, locally Lipschitz random function. As the gradient may not exist, it is replaced by a certain operator: a reasonable choice is to use an element of the Clarke subdifferential of the random function; an other choice is the output of the celebrated backpropagation algorithm, which is popular amongst practionners, and whose properties have recently been studied by Bolte and Pauwels [7]. Since the expectation of the chosen operator is not in general an element of the Clarke subdifferential BF of the mean function, it has been assumed in the literature that an oracle of BF is available. As a first result, it is shown in this paper that such an oracle is not needed for almost all initialization points of the algorithm. Next, in the small step size regime, it is shown that the interpolated trajectory of the algorithm converges in probability (in the compact convergence sense) towards the set of solutions of the differential inclusion. Finally, viewing the iterates as a Markov chain whose transition kernel is indexed by the step size, it is shown that the invariant distribution of the kernel converge weakly to the set of invariant distribution of this differential inclusion as the step size tends to zero. These results show that when the step size is small, with large probability, the iterates eventually lie in a neighborhood of the critical points of the mean function F .",
  "timestamp": "2026-01-08T22:06:41.544Z",
  "rating": "novote",
  "publishedDate": "2020-05-18",
  "tags": [
    "math.NA",
    "math.OC"
  ],
  "doi": "",
  "journalName": "",
  "sourceType": "url",
  "arxiv_tags": [
    "math.NA",
    "math.OC"
  ]
}

**Comment by mbahng** [comment]
{
  "_data": {
    "sourceId": "arxiv",
    "paperId": "2005.08513",
    "url": "https://arxiv.org/pdf/2005.08513",
    "title": "",
    "authors": "",
    "abstract": "",
    "timestamp": "2026-01-08T22:06:41.544Z",
    "rating": "novote",
    "publishedDate": "",
    "tags": [],
    "doi": "",
    "journalName": "",
    "sourceType": "url"
  },
  "_meta": {
    "client_version": "0.11.1",
    "timestamp": "2026-01-08T22:06:42.036Z",
    "update_mode": "append",
    "issue_number": 66
  },
  "type": "initial_state"
}
